.gitignore

```diff
@@ -1,4 +1,4 @@
-node_modules
-tmp
 .DS_Store
-
+/node_modules/
```

Comment:
The comment you made for grunt is moot :P

As for this, I guess it's a personal taste. I'll change this one.